maven-settings.xml

```diff
@@ -0,0 +1,9 @@
+<settings>
+  <mirrors>
+    <mirror>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/public/</url>
```

Comment:
This is used in a separate `mvn` invocation that downloads dependencies of core and all plugins before scanning for extension implementations (perhaps to make it compilable, unclear to me).

My current best guess is that without this, the standalone `mvn` invocation trying to download `jenkins-core` dependencies doesn't find the parent pom that's not in the parent dir.